velox/common/caching/SsdFile.cpp

```diff
@@ -474,6 +480,15 @@ void SsdFile::updateStats(SsdCacheStats& stats) const {
   for (auto pins : regionPins_) {
     stats.numPins += pins;
   }
+
+  stats.errorsOpenFile += stats_.errorsOpenFile;
```

Comment:
Please be consistent in naming and let remove SSD in stats naming.

Reply:
We have errors from opening ssd file, the checkpoint file and log file. So I think we'd be better to keep 'ssd', 'checkpoint' and 'log' to tell them apart.